src/dataObjects/dtos/singleRights/ResourceList.tsx

```diff
@@ -1,5 +1,5 @@
 /**
- * Represents the delegation input data transfer object.
+ * Represents a list of Resources that's sent in to backend to enable backend to identify the resource.
```

Comment:
Does that mean we could have asked for the statuses of multiple different service resources at once?

Reply:
Think so but not completely sure, ask Jon Kjetil

Reply:
He says you can only check one service resource at a time, so I think this type name and description is a bit misleading 😅

Reply:
Ahh alright then

Reply:
But you should suggest a name when you think it's misleading 😉

Reply:
Even though the list contains only one item it is still a list

Reply:
But I don't fully understand what the object is myself or why its even a list if it is only meant to represent only one object 😅

Is the one object defined by the list items? Is it just a way to wrap the one item? (if so why?)

Reply:
Oh yeah, it was a list of id-value pairs (either one or two depending on if the resource belongs to the registry, A2 or A3)

So, I guess the list itself is a resource identifier(s). The identifier(s) used to identify a singular resource

So... ResourceIdentifierDTO?

Reply:
good suggestion, changing it again